src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx

```diff
@@ -126,9 +130,12 @@ class PaymentInformationEditModal extends React.Component {
 
           <ErrorableSelect
             label="Account type"
+            name="account-type"
             value={accountType.value}
+            errorMessage={accountType.errorMessage}
             onValueChange={this.onAccountTypeChanged}
             options={Object.values(ACCOUNT_TYPES_OPTIONS)}
+            includeBlankOption
```

Comment:
To this file, there were a couple changes -

1. Added a blank option for the account-type (savings or checking) and validation for when that field is empty
2. Added a component, `PaymentInformationEditModalError`, for rendering errors 
3. Copy updates